### PR TITLE
Fix typo, fix first responder check, moved default hunting zone

### LINKED ISSUE #140: Bug: Checking for IsDispatchJob which does not exist
In sv_main.lua there is a check for `IsDispatchJobs`. This variable no longer exists, it was removed in this PR: https://github.com/Project-Sloth/ps-dispatch/pull/126/files . 

```
RegisterCommand('togglealerts', function(source, args, user)
	local source = source
	if IsDispatchJob then
		TriggerClientEvent('dispatch:manageNotifs', source, args[1])
	end
end)
```

Looks like `Config.AuthorizedJobs.FirstResponder.Check(Player.PlayerData)` may be the updated check that should be used here instead? @DonHulieo, does this seem like the correct check (asking since you made the above changes)?

### PR DESCRIPTION
**Pull request by BackSH00TER**
- Fixed typo from `InjuriedPerson` to `InjuredPerson`
- Moved default hunting zone from middle of LSIA to in the county by the river
  
image

- Fixed the first responder check in `togglealerts`
   - Fixes https://github.com/Project-Sloth/ps-dispatch/issues/140

### CODE REVIEW

**Review by ADRNALN — APPROVED** [review]

File: server/sv_main.lua [hunk 1]
```diff
@@ -87,9 +87,10 @@ end)
...
 
 RegisterCommand('togglealerts', function(source, args, user)
-	local source = source
-	if IsDispatchJob then
-		TriggerClientEvent('dispatch:manageNotifs', source, args[1])
+	local src = source
+    local Player = QBCore.Functions.GetPlayer(src)
+	if Config.AuthorizedJobs.FirstResponder.Check(Player.PlayerData) then
+		TriggerClientEvent('dispatch:manageNotifs', src, args[1])
 	end
```
Comment (Z3rio) [inline server/sv_main.lua]:
```suggestion
    local src = source
    local Player = QBCore.Functions.GetPlayer(src)
    if Config.AuthorizedJobs.FirstResponder.Check(Player.PlayerData) then
        TriggerClientEvent('dispatch:manageNotifs', src, args[1])
    end
```

resolve linting

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Do not create issues on GitHub if you need help. Issues are for bug reporting an
 
 - exports['ps-dispatch']:Fight()
 
-- exports['ps-dispatch']:InjuriedPerson()
+- exports['ps-dispatch']:InjuredPerson()
```
Comment (xFutte) [inline README.md:28]: This breaks backward compitability
```suggestion
- exports['ps-dispatch']:InjuriedPerson()
```

File: client/cl_eventhandlers.lua [hunk 3]
```diff
@@ -190,7 +190,7 @@ AddEventHandler('gameEventTriggered', function(name, args)
     -- If the victim isn't dead, don't trigger the event as well
     if not isDead then Config.Timer['PlayerDowned'] = Config.PlayerDowned.Fail return end
     -- Check if the player is and EMS, trigger the correct down alerts
-    if not Config.AuthorizedJobs.FirstResponder.Check() then exports['ps-dispatch']:InjuriedPerson() end
+    if not Config.AuthorizedJobs.FirstResponder.Check() then exports['ps-dispatch']:InjuredPerson() end
```
Comment (xFutte) [inline client/cl_eventhandlers.lua:193]: Breaks backward compatibility. Revert.
```suggestion
    if not Config.AuthorizedJobs.FirstResponder.Check() then exports['ps-dispatch']:InjuriedPerson() end
```

File: client/cl_events.lua [hunk 4]
```diff
@@ -199,7 +199,7 @@ end
 
 exports('Fight', Fight)
 
-local function InjuriedPerson()
+local function InjuredPerson()
```
Comment (xFutte) [inline client/cl_events.lua:202]:
```suggestion
local function InjuriedPerson()

```

File: client/cl_events.lua [hunk 5]
```diff
@@ -223,7 +223,7 @@ local function InjuriedPerson()
     })
 end
 
-exports('InjuriedPerson', InjuriedPerson)
+exports('InjuredPerson', InjuredPerson)
```
Comment (xFutte) [inline client/cl_events.lua:226]:
```suggestion
exports('InjuriedPerson', InjuriedPerson)

```

**Review by xFutte — CHANGES_REQUESTED** [review]

**Comment by xFutte** [comment]
Changing the export name will break backward compatibility. Since this resource has been around for a while, we can't simply do this change in a minor version. If we wish to do so, we need to make a new major release.

**Comment by adamdrakon** [comment]
> Changing the export name will break backward compatibility. Since this resource has been around for a while, we can't simply do this change in a minor version. If we wish to do so, we need to make a new major release.

Not hugely important, but could add `exports('InjuredPerson', InjuriedPerson)`, but then, so too could anyone else add it on their own end.

**Pull request closed without merging**